dev/src/main/java/org/eclipse/codewind/intellij/ui/tree/CodewindTreeNodeCellRenderer.java

```diff
@@ -134,20 +134,25 @@ private String getText(CodewindApplication application) {
 
         if (application.isEnabled()) {
             AppStatus appStatus = application.getAppStatus();
-            if (appStatus != AppStatus.UNKNOWN) {
+            BuildStatus buildStatus = application.getBuildStatus();
+
+            if (appStatus != AppStatus.UNKNOWN || buildStatus == BuildStatus.UNKOWN) {
+                // If both app status and build status are unknown, add the 'status unknown' message.
+                // else if app status is not unknown, add it's status message.
```

Comment:
Hey John, could you clarify the comments on 140 and 141?  In the new if statement, I'm reading it as:
"If the app status is not unknown or the build status is unknown, then show the application start mode (eg. Running or Debugging)."

Then the if statement below, it is simply whether the build status is not unknown, which will then show the known build status.

Reply:
Rather than clarify the comment, I've clarified the logic -- the behaviour is the same, but it is hopefully easier to understand.